Samples/src/H2SimulationCmdLine/Operation.qs

```diff
@@ -30,61 +30,61 @@ namespace Microsoft.Quantum.Samples.H2Simulation {
-        set bondCoefficients[18] = [0.2565, -0.2036, 0.5059, 0.1018, 0.1018];
-        set bondCoefficients[19] = [0.2467, -0.181, 0.4974, 0.1034, 0.1034];
-        set bondCoefficients[20] = [0.2374, -0.1603, 0.4892, 0.105, 0.105];
-        set bondCoefficients[21] = [0.2286, -0.1413, 0.4812, 0.1067, 0.1067];
-        set bondCoefficients[22] = [0.2203, -0.1238, 0.4735, 0.1083, 0.1083];
-        set bondCoefficients[23] = [0.2123, -0.1077, 0.466, 0.11, 0.11];
-        set bondCoefficients[24] = [0.2048, -0.0929, 0.4588, 0.1116, 0.1116];
-        set bondCoefficients[25] = [0.1976, -0.0792, 0.4518, 0.1133, 0.1133];
-        set bondCoefficients[26] = [0.1908, -0.0666, 0.4451, 0.1149, 0.1149];
-        set bondCoefficients[27] = [0.1843, -0.0549, 0.4386, 0.1165, 0.1165];
-        set bondCoefficients[28] = [0.1782, -0.0442, 0.4323, 0.1181, 0.1181];
-        set bondCoefficients[29] = [0.1723, -0.0342, 0.4262, 0.1196, 0.1196];
-        set bondCoefficients[30] = [0.1667, -0.0251, 0.4204, 0.1211, 0.1211];
-        set bondCoefficients[31] = [0.1615, -0.0166, 0.4148, 0.1226, 0.1226];
-        set bondCoefficients[32] = [0.1565, -0.0088, 0.4094, 0.1241, 0.1241];
-        set bondCoefficients[33] = [0.1517, -0.0015, 0.4042, 0.1256, 0.1256];
-        set bondCoefficients[34] = [0.1472, 0.0052, 0.3992, 0.127, 0.127];
-        set bondCoefficients[35] = [0.143, 0.0114, 0.3944, 0.1284, 0.1284];
-        set bondCoefficients[36] = [0.139, 0.0171, 0.3898, 0.1297, 0.1297];
-        set bondCoefficients[37] = [0.1352, 0.0223, 0.3853, 0.131, 0.131];
-        set bondCoefficients[38] = [0.1316, 0.0272, 0.3811, 0.1323, 0.1323];
-        set bondCoefficients[39] = [0.1282, 0.0317, 0.3769, 0.1335, 0.1335];
-        set bondCoefficients[40] = [0.1251, 0.0359, 0.373, 0.1347, 0.1347];
-        set bondCoefficients[41] = [0.1221, 0.0397, 0.3692, 0.1359, 0.1359];
-        set bondCoefficients[42] = [0.1193, 0.0432, 0.3655, 0.137, 0.137];
-        set bondCoefficients[43] = [0.1167, 0.0465, 0.362, 0.1381, 0.1381];
-        set bondCoefficients[44] = [0.1142, 0.0495, 0.3586, 0.1392, 0.1392];
-        set bondCoefficients[45] = [0.1119, 0.0523, 0.3553, 0.1402, 0.1402];
-        set bondCoefficients[46] = [0.1098, 0.0549, 0.3521, 0.1412, 0.1412];
-        set bondCoefficients[47] = [0.1078, 0.0572, 0.3491, 0.1422, 0.1422];
-        set bondCoefficients[48] = [0.1059, 0.0594, 0.3461, 0.1432, 0.1432];
-        set bondCoefficients[49] = [0.1042, 0.0614, 0.3433, 0.1441, 0.1441];
-        set bondCoefficients[50] = [0.1026, 0.0632, 0.3406, 0.145, 0.145];
-        set bondCoefficients[51] = [0.1011, 0.0649, 0.3379, 0.1458, 0.1458];
-        set bondCoefficients[52] = [0.0997, 0.0665, 0.3354, 0.1467, 0.1467];
-        set bondCoefficients[53] = [0.0984, 0.0679, 0.3329, 0.1475, 0.1475];
+        let bondCoefficients = new Double[][nBondLengths]
+            w/ 0 <- [0.5678, -1.4508, 0.6799, 0.0791, 0.0791]
```

Comment:
Why not just use an array-of-array literal here(for the entire bondCoefficients )? The use of mutability seems rather unnecessary.

Reply:
Agreed, `mutable` isn't needed here. My understanding was that this design was meant to emphasize the indices for each element, but I'll convert to a literal.